src/sidebar/store/modules/groups.js

```diff
@@ -66,6 +73,17 @@ const reducers = {
       groups: [],
     };
   },
+
+  CHANGE_FOCUS_MODE_USER: function (state, action) {
```

Comment:
If you add `/** @type {import('../create-store').ReducerMap<State> */` above the `const reducers` declaration and `/** @param {{ user: ... }} action */` above the reducer, this will type the `state` and `action` properties. We can work on this as we fix the remaining missing types in the client next year.